src/components/App/app.view.js

```diff
@@ -1,19 +1,23 @@
 import { Switch, Route } from "react-router-dom";
+import { Container } from "react-bootstrap";
 import AppBehavior from "./app.behavior.js";
 import GameDetails from "../GameDetails/game.details.view.js";
 import SearchBar from "../SearchBar/search.bar.view.js";
 import TopGamesTable from "../TopGamesTable/top.games.table.view.js";
+import "./app.css";
 import "../../assets/global.css";
 
 const App = () => {
   const [topTenGames, tableOptions] = AppBehavior();
 
   return (
-    <div>
+    <div className="main-content">
```

Comment:
Consider using bootstrap spacing utilities: https://getbootstrap.com/docs/5.2/layout/utilities/#margin-and-padding: `pt-3`

Bootstrap also uses relative values instead of pixels. If you can't use bootstrap consider switching from 54px to an `em` equivalent.